lib/openshift/flakes/route_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'base_helper'
+
+module BushSlicer
+
+  # this class should help with parsing route specifications
+  class RouteSpec
+    include Common::Helper
+
+    attr_reader :struct
+    private :struct
+
+    def initialize(struct)
+      @struct = struct
+    end
+
+    module ExportMethods
+      def  host
+        return struct['host']
+      end
+
+      def targetPort
+        return struct.dig('port', 'targetPort')
+      end
+
+      def subdomain
+        return struct['subdomain']
+      end
+
+      def tls_insecureEdgeTerminationPolicy
+        return struct.dig('tls', 'insecureEdgeTerminationPolicy')
+      end
+
+      def tls_termination
+        return struct.dig('tls', 'termination')
+      end
+
+      def to_kind
```

Comment:
maybe `target_kind` instead of `to_kind`?

Reply:
Anyway they still looks good if we don't change

Reply:
we can probably alias the methods, wdyt?